src/components/author-primitives/packer/plugin-badge/index.tsx

```diff
@@ -18,34 +21,30 @@ type PluginLabelType =
 const badgeTypes = {
 	official: {
 		label: 'Official',
-		theme: 'gold',
-		iconSvg: svgRibbonIcon,
+		iconSvg: <IconHashicorp16 />,
 	},
 	community: {
 		label: 'Community',
-		theme: 'gray',
-		iconSvg: false,
+		iconSvg: <IconUsers16 />,
 	},
 	hcp_packer_ready: {
 		label: 'HCP Packer Ready',
-		theme: 'blue',
-		iconSvg: svgCheckIcon,
+		iconSvg: <IconRocket16 />,
 	},
 	verified: {
 		label: 'Verified',
-		theme: 'purple',
-		iconSvg: svgRibbonIcon,
+
+		iconSvg: <IconAward16 />,
 	},
 	archived: {
 		label: 'Archived',
-		theme: 'light-gray',
-		iconSvg: false,
+		iconSvg: <IconArchive16 />,
 	},
 }
 
 function PluginBadge({ type }: { type: PluginLabelType }): React.ReactElement {
-	const { label, theme, iconSvg } = badgeTypes[type]
-	return <Badge label={label} theme={theme as BadgeTheme} iconSvg={iconSvg} />
```

Comment:
TODO: need to follow up to see other use cases of this true legacy badge component.